frontend/lib/src/components/elements/ArrowVegaLiteChart/styled-components.ts

```diff
@@ -14,7 +14,75 @@
  * limitations under the License.
  */
 
+import { CSSObject } from "@emotion/react"
 import styled from "@emotion/styled"
+import { transparentize } from "color2k"
+
+import { EmotionTheme } from "@streamlit/lib/src/theme"
+
+export const StyledVegaLiteChartTooltips = (
+  theme: EmotionTheme
+): CSSObject => ({
+  "#vg-tooltip-element": {
+    visibility: "hidden",
+    position: "fixed",
+    fontFamily: theme.genericFonts.bodyFont,
+    color: theme.colors.bodyText,
+    border: `${theme.sizes.borderWidth} solid ${theme.colors.borderColor}`,
+    backgroundColor: transparentize(theme.colors.bgColor, 0.05),
+    fontSize: theme.fontSizes.twoSm,
+    boxShadow: "rgb(0 0 0 / 16%) 0px 1px 4px",
+    padding: `${theme.spacing.xs} ${theme.spacing.md}`,
+    borderRadius: theme.radii.default,
```

Comment:
it looks like the border-radius before was `3px`, which would be closer to `theme.radii.sm` with `0.125rem`